node/npm/glide/index.ts

```diff
@@ -85,6 +106,13 @@ function initialize() {
         ConnectionError,
         ClusterTransaction,
         Transaction,
+        createLeakedArray,
```

Comment:
why?

Reply:
As the other export, we need it to be available. test failing on it.